CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# v1.0.0
+
+## Breaking Changes
+
+- Deleted `/api/v1/epochs/<number>`
+- Deleted `/api/v1/epochs/missing` 
+- Deleted `/api/v1/score/health`
+- Deleted `/api/v1/score`
+- Deleted `/api/v1/validators/balance`
+- Deleted `/api/v1/validators/import`
+- Removed `only-active` query param from `/api/v1/validators`. 
```

Comment:
Any reason to remove this query param?

Reply:
we are not using it anywhere, and makes logic a bit more complicated.